src/ldw-barsPlus.js

```diff
@@ -259,17 +260,18 @@ export default {
         : (p.indexOf(a.key) > p.indexOf(b.key) ? 1 : 0));
     });
     n.forEach(function (d, idx) {
-      var t = 0, v = [], j = 0, num, txt;
+      var t = 0, v = [], j = 0, num, txt, mNum;
       for (var i = 0; i < q.length; i++) {
         let elm;
-        if (d.values.length <= j || d.values[j].key != q[i]) {
+        if (d.values.length <= j || d.values[j].values[0][1].qText != q[i]) {
           num = 0;
           txt = "-";
           elm = [];
         }
         else {
           num = d.values[j].values[0][2].qNum;
           txt = d.values[j].values[0][2].qText;
+          mNum = d.values[j].values[0][1].qNum;
```

Comment:
`d.values[j].values[0][1]` when it starts getting this complex an intermediary variable or two might be a good idea, in this case most of it is also duplicated a couple of times

Reply:
tried that but it didn't work
i tried having a variable for `d.values[j].values[0]`
and then 3 variables for the 3 rows `d.values[j].values[0][0]` `d.values[j].values[0][1]` `d.values[j].values[0][2]`
also didn't work
any suggestion for that

Reply:
@kristofferlind